pkg/tenant/tenant.go

```diff
@@ -75,30 +75,30 @@ func (s *Server) Setup() (err error) {
 	// Configures Sentry
 	if s.conf.Sentry.UseSentry() {
 		if err = sentry.Init(s.conf.Sentry); err != nil {
-			return err
+			return fmt.Errorf("could not init sentry: %w", err)
 		}
 	}
 
 	// Connect to services when not in maintenance mode
 	if !s.conf.Maintenance {
 		// Connect to the trtl database
 		if err = db.Connect(s.conf.Database); err != nil {
-			return err
+			return fmt.Errorf("could not connect to db: %w", err)
 		}
 
 		// Connect to Ensign
 		if s.ensign, err = s.conf.Ensign.Client(); err != nil {
-			return err
+			return fmt.Errorf("could not create ensign client: %w", err)
```

Comment:
Added these error messages to help figure out why the server was crashing in tests; it helped me discover that this was the line of code that was causing the problem.